backend/src/src-wsimport/src/main/java/org/eclipse/sw360/wsimport/rest/WsImportService.java

```diff
@@ -70,4 +75,20 @@ public WsProject getWsProject(String projectToken, TokenCredentials tokenCredent
 
     }
 
+    public  WsProjectVitalInformation[] getOrganizationalProjectVitals(TokenCredentials tokenCredentials) throws JsonSyntaxException {
+        String projectVitalString;
+        try {
+            projectVitalString = restClient.getData(REQUEST__GET_ORGANIZATION_PROJECT_VITALS, tokenCredentials.getToken(), WsTokenType.ORGANIZATION, tokenCredentials);
+        } catch (IOException ioe) {
+            LOGGER.error(ioe);
+            return null;
+        }
+        WsProjectVitals wsProjectVitals = gson.fromJson(projectVitalString, WsProjectVitals.class);
+        if (wsProjectVitals.getProjectVitals() != null) {
+            return wsProjectVitals.getProjectVitals();
+        } else {
+            return null;
```

Comment:
Maybe add some logs in the else if condition like logger.info or logger.trace
"Project vitals are empty...."

Reply:
I will add logging